tests/memberclient.py

```diff
@@ -74,6 +74,31 @@ def test_update_recovery_shares(network, args):
     return network
 
 
+@reqs.description("Send an unsigned request where signature is required")
+def test_missing_signature(network, args):
```

Comment:
This is kind of already tested in `governance_history.py` when submitting unsigned votes... but this is muh better! Could we remove the test case in `governance_history.py` and also remove the `force_unsigned` altogether?

Reply:
Sure, I've removed the similar bit from `governance_history.py`.